models/src/main/java/com/connexta/transformation/rest/models/TransformRequest.java

```diff
@@ -17,10 +17,14 @@
 import io.swagger.annotations.ApiModelProperty;
 import java.net.URL;
 import java.util.Objects;
+import java.util.UUID;
 import javax.validation.constraints.NotNull;
 
 /** TransformRequest */
 public class TransformRequest {
+  @JsonProperty("datasetId")
```

Comment:
✏️ `datasetUUID` might be a better name for this property

Reply:
It is an identifier that uses the UUID format. We'll keep it as id.